From a251de1e43c62963e33010c49c69d7aaa0bedac3 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 07 Oct 2018 11:49:30 +0300
Subject: [PATCH] Makefile: version: align the embedded version

Embedded version is not aligned with the rest of the s/w versions.
Deal with the problem by embedding the general s/w version inside the
embedded f/w version.

Resolves SW-2065.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/Makefile b/Makefile
index 92c4e37..5a58c2f 100644
--- a/Makefile
+++ b/Makefile
@@ -75,12 +75,13 @@
 	grep -oE "origin/v[0-9]+\.[0-9]+\.[0-9]+" | \
 	sed 's/origin\///' | sort -n | tail -n 1)
 
+RELEASE_VER_COMPAT := $(patsubst v%,%,$(RELEASE_BRANCH))
 RELEASE_JOB_BRANCH := "package_branch=$(RELEASE_BRANCH)"
-RELEASE_VER_COMPAT := "release_version=$(patsubst v%,%,$(RELEASE_BRANCH))"
+RELEASE_JOB_VER_COMPAT := "release_version=$(RELEASE_VER_COMPAT)"
 RELEASE_JOB_URL := http://192.168.0.4:8080/job/goya-release-test/buildWithParameters
 RELEASE_JOB_CAUSE := "cause=Started_by_$(shell whoami)"
 RELEASE_JOB_TOKEN := "token=goya-release"
-RELEASE_JOB_PARAMS := $(RELEASE_JOB_TOKEN)&$(RELEASE_JOB_CAUSE)&$(RELEASE_JOB_BRANCH)&$(RELEASE_VER_COMPAT)
+RELEASE_JOB_PARAMS := $(RELEASE_JOB_TOKEN)&$(RELEASE_JOB_CAUSE)&$(RELEASE_JOB_BRANCH)&$(RELEASE_JOB_VER_COMPAT)
 
 define NO_RELEASE_BRANCH_ERROR =
 Assuming the upstream release branch looks like 'origin/vx.y.z'.
@@ -109,7 +110,7 @@
 
 .PHONY: gaudi-release goya-release
 gaudi-release goya-release: hl-check-release hl-check-branch
-	$(eval EMBD_FULL_VER=hl-$(patsubst %-release,%,$@)-$(EMBD_RELEASE_VER))
+	$(eval EMBD_FULL_VER=hl-$(patsubst %-release,%,$@)-$(RELEASE_VER_COMPAT)-fw-$(EMBD_RELEASE_VER))
 	$(eval RELEASE_JOB_PROD_NAME="product_name=$(patsubst %-release,%,$@)")
 	$(eval RELEASE_JOB_TAG_PTR="embedded_release_tag=$(EMBD_FULL_VER)")
 	$(eval RELEASE_JOB_PARAMS=$(RELEASE_JOB_PARAMS)&$(RELEASE_JOB_PROD_NAME)&$(RELEASE_JOB_TAG_PTR))
